adminsortable/admin.py

```diff
@@ -192,6 +193,9 @@ def sort_view(self, request):
         except AttributeError:
             verbose_name_plural = opts.verbose_name_plural
 
+        csrf_token = get_token(self.request) if \

```

Comment:
I would probably write this as:
```
csrf_token = ''

if getattr(settings, 'CSRF_USE_SESSIONS', False):
    csrf_token = get_token(self.request)
```

This way you don't have to have the else and have the hanging `if`